src/translations/en.json

```diff
@@ -92,8 +92,23 @@
     "sign_in.password.placeholder": "Password",
     "sign_in.username.placeholder": "Username",
 
+    "shipment.add.header": "New Shipment",
+    "shipment.edit.header": "Edit Shipment",
     "shipment_details": "Shipment",
     "shipment.header": "Shipment",
+    "shipment.label.shipment_number": "Shipment Number",
+    "shipment.label.transporter_name": "Transporter Name",
+    "shipment.label.driver_name": "Driver Name",
+    "shipment.label.vehicle_number": "Vehicle Number",
+    "shipment.label.ship_date": "Shipment Date (DD/MM/YYYY)",
+    "shipment.label.expected_delivery_date": "Expected Delivery Date (DD/MM/YYYY)",
+    "shipment.label.custom_route_number": "Custom Route Number",
+    "shipment.label.delay_justified": "Delay Justified",
+    "shipment.label.delivery_date": "Delivery Date (DD/MM/YYYY)",
+    "shipment.label.delivery_status": "Delivery Status",
+    "shipment.label.pod_status": "e-POD Status",
+    "shipment.label.pod": "Paper Based POD Received",
+    "shipment.label.customer_name": "Customer",
```

Comment:
Here I think it's better to write keys similar to already existing keys, so instead of `shipment.label.customer_name` -> `shipment.customer_name.label`